hydroflow_plus/src/builder.rs

```diff
@@ -105,12 +101,16 @@ impl<'a, D: LocalDeploy<'a>> FlowBuilder<'a, D> {
 
 #[derive(Clone)]
 pub struct BuiltFlow<'a, D: LocalDeploy<'a>> {
-    pub ir: Vec<HfPlusLeaf>,
+    pub(crate) ir: Vec<HfPlusLeaf>,
 
     _phantom: PhantomData<&'a mut &'a D>,
 }
 
 impl<'a, D: LocalDeploy<'a>> BuiltFlow<'a, D> {
+    pub fn ir(&self) -> &Vec<HfPlusLeaf> {
```

Comment:
kinda expect clippy to want `&[HfPlusLeaf]` but I guess that's just in argument position? Doesn't really matter